run.sh

```diff
@@ -45,21 +45,30 @@ fi
 # Set group users to specified group id (non unique)
 groupmod -o -g "${PGID}" unifi-video &>/dev/null
 
+# Create logs directory
+mkdir -p /var/lib/unifi-video/logs
```

Comment:
Shouldn't this be gated on the directory not already existing?

Reply:
I considered that, but w/ `mkdir` it just really isn't worth bothering with. If the directory exists, it just won't do anything. If it doesn't, it'll make the full path needed to make it. I didn't see any reason to over complicate the script.